sklearn/preprocessing/_function_transformer.py

```diff
@@ -72,18 +78,33 @@ class FunctionTransformer(BaseEstimator, TransformerMixin):
         Dictionary of additional keyword arguments to pass to inverse_func.
 
     """
-    def __init__(self, func=None, inverse_func=None, validate=True,
-                 accept_sparse=False, pass_y='deprecated', check_inverse=True,
-                 kw_args=None, inv_kw_args=None):
+    def __init__(self, func=None, inverse_func=None, validate=None,
+                 accept_sparse=False, force_all_finite=True,
+                 pass_y='deprecated', check_inverse=True, kw_args=None,
+                 inv_kw_args=None):
         self.func = func
         self.inverse_func = inverse_func
         self.validate = validate
         self.accept_sparse = accept_sparse
+        self.force_all_finite = force_all_finite
```

Comment:
What's the purpose of it? Seems that force_all_finite is not used?

Reply:
good catch, it is a remaining from another option